src/models/data.py

```diff
@@ -269,7 +259,7 @@ def _calculate_aggs(self, x: xr.Dataset) -> np.ndarray:
 
         if (self.normalizing_dict is not None) and (self.normalizing_array is None):
             self.normalizing_array_ym = self.calculate_normalizing_array(
-                list(yearly_mean.data_vars), yearly_agg=True)
```

Comment:
we're removing the `yearly_agg` because we now capture that using the normalization defined across all timesteps fed into the model in the `X` data right?

Reply:
also would `yearly_agg` make sense if we don't include 12 previous months? (say 3 prior months)

Reply:
We're removing the `yearly_agg` argument from the function, because now there doesn't need to be a distinction. However, the `yearly_agg` will still be there

Reply:
hmm so the yearly aggregation is still sent to the model even if we define our `X.nc`  in the engineer as 3 timesteps?

Reply:
yea - yearly aggregation may be a misnomer, since the aggregation is calculated from the input data.

So if the engineer produces 3 timesteps, then the aggregation is accross 3 timesteps

Reply:
cool!
merge this bad boy